kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1148,6 +1148,19 @@ object KyuubiConf {
       .checkValues(OperationModes.values.map(_.toString))
       .createWithDefault(OperationModes.NONE.toString)
 
+  val OPERATION_PLAN_ONLY_EXCLUDES: ConfigEntry[Seq[String]] =
+    buildConf("operation.plan.only.excludes")
+      .doc("Comma-separated list of query plan names, in the form of simple class names, i.e, " +
+        "for `set abc=xyz`, the value will be `SetCommand`. For those auxiliary plans, such as " +
+        "`switch databases`, `set properties`, or `create temporary view` e.t.c, " +
+        "which are used for setup evaluating environments for analyzing actual queries, " +
+        "we can use this config to exclude them and let them take effect. " +
+        s"See also ${OPERATION_PLAN_ONLY_MODE.key}.")
+      .version("1.5.0")
+      .stringConf
+      .toSequence()
+      .createWithDefault(Seq("ResetCommand", "SetCommand", "SetNamespaceCommand", "UseStatement"))
```

Comment:
IMO, the configuration here is to provide the exact className, if you want to skip cacheTable,
`CacheTableStatement` in spark 3.0 and `CacheTableCommand` in Spark 3.2, it's too hard for users to set the right options. Could the options here be more generic, like set/reset/cacheTable/CreateTempView, rather than requiring the actual class name?

Reply:
@iodone, the point above doesn't make sense. The engine side couldn't do any mapping for the user-defined options.